hack/set-odh-release.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+set -o errexit
```

Comment:
Just curious, what do you think about having this in the toolbox repo?

Reply:
I don't mind having it there, but to me it seems this script is very limited in it's context. I feel like it doesn't make much sense to have it as a generic script if it's usable in a single repo context. I feel the same about the `onboarding.sh` script.

https://github.com/operate-first/toolbox/issues/10#issuecomment-787993230

Reply:
I don't think I disagree. Only counter-point is that this is applicable to any deployments of the ODH, so it's arguably somewhat general. 

Maybe for the time being we leave it here, and if there's a situation where we would add this to any other operate-first repo we can move it over.